### PR TITLE
Backup: Rename plugin to Jetpack VaultPress Backup

### PR DESCRIPTION
**Pull request by Initsogar**
#### Changes proposed in this Pull Request:

* Update `readme.txt` to rename Jetpack Backup -> Jetpack VaultPress Backup.
* Adding myself (`initsogar`) as a contributor.

#### Does this pull request change what data or activity we track or use?
No

#### Other information:

- [ ] ~Have you written new tests for your changes, if applicable?~
- [ ] Have you checked the E2E test CI results, and verified that your changes do not break them?

#### Jetpack product discussion

* p1HpG7-j1R-p2
* 1202586245449203-as-1203348579665815/f

### CODE REVIEW

File: projects/plugins/backup/readme.txt [hunk 1]
```diff
@@ -8,11 +8,11 @@ Stable tag: 1.2.0
 License: GPLv2 or later
 License URI: http://www.gnu.org/licenses/gpl-2.0.html
 
-Jetpack is the most proven WordPress backup plugin with over 270 million backups. Save each change and get back online fast with one‑click restores.
+VaultPress is the most proven WordPress backup plugin with over 270 million backups. Save each change and get back online fast with one-click restores.
 
 == Description ==
 
-Jetpack Backup is a new plugin that focuses specifically on Jetpack's Backup features.
+Jetpack VaultPress Backup is a plugin that delivers backups and restores without the other parts of Jetpack.
```
Comment (macbre) [inline projects/plugins/backup/readme.txt:15]: Should someone review these copy changes in the plugin description? Or was it already reviewed / proposed in a separate ticket?

**Review by macbre — DISMISSED** [review]
LGTM, no tpyos found 😉

File: projects/plugins/backup/readme.txt [hunk 2]
```diff
@@ -1,5 +1,5 @@
-=== Jetpack Backup ===
-Contributors: automattic, bjorsch, fgiannar, jeherve, jwebbdev, kraftbj, macbre, samiff, sermitr, williamvianas
+=== Jetpack VaultPress Backup ===
+Contributors: automattic, bjorsch, fgiannar, jeherve, jwebbdev, kraftbj, macbre, samiff, sermitr, williamvianas, initsogar
```
Comment (macbre) [inline projects/plugins/backup/readme.txt]: 👍🏻

File: projects/plugins/backup/readme.txt (hunk 1 above)
Comment (Initsogar) [reply projects/plugins/backup/readme.txt:15]: @macbre Yes, it was proposed in the doc shared here: p1HpG7-j1R-p2

**Review by macbre — APPROVED** [review]

**Pull request merged**